rules-reviewed/camel3/camel2/xml-dsl-changes.windup.xml

```diff
@@ -0,0 +1,260 @@
+                        <replacement>keyOrTrustStoreParametersId</replacement>
+                        <search>keyOrTrustStoreParametersId</search>
+                    </quickfix>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="xml-dsl-changes-00005">
+            <when>
+                <or>
+                    <xmlfile matches="//b:hystrix">
+                        <namespace prefix="b" uri="http://camel.apache.org/schema/blueprint"/>
+                    </xmlfile>
+                    <xmlfile matches="//*/c:hystrix">
+                        <namespace prefix="c" uri="http://camel.apache.org/schema/spring"/>
+                    </xmlfile>
+                </or>
+            </when>
+            <perform>
+                <hint title="`hystrix` was renamed" effort="1"
+                      category-id="mandatory">
+                    <message>Hystrix circuit breaker has been generalized as circuit breaker. Use `circuitBreaker`
+                        instead of `hystrix'.
+                    </message>
+                    <link href="https://camel.apache.org/manual/latest/camel-3-migration-guide.html#_xml_dsl_migration"
+                          title="Camel 3 - Migration Guide"/>
+                    <quickfix type="REPLACE" name="camel.eip.hystrix.replacement">
+                        <replacement>circuitBreaker</replacement>
+                        <search>hystrix</search>
+                    </quickfix>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="xml-dsl-changes-00006">
+            <when>
+                <or>
+                    <xmlfile matches="//*/b:aggregate/b:completionSize">
+                        <namespace prefix="b" uri="http://camel.apache.org/schema/blueprint"/>
+                    </xmlfile>
+                    <xmlfile matches="//*/c:aggregate/c:completionSize">
+                        <namespace prefix="c" uri="http://camel.apache.org/schema/spring"/>
+                    </xmlfile>
+                </or>
+            </when>
+            <perform>
+                <hint title="`completionSize` in aggregate was renamed" effort="1"
+                      category-id="mandatory">
+                    <message>Use
+                        Use `completionSizeExpression` instead of `completionSize`. Expressions (not the attributes) for
+                        setting correlation size/timeout were renamed.
+                    </message>
+                    <link href="https://camel.apache.org/manual/latest/camel-3-migration-guide.html#_aggregate_eip_in_xml_dsl"
+                          title="Camel 3 - Migration Guide - Aggregate EIP changes"/>
+                    <quickfix type="REPLACE" name="camel.eip.aggregate.size.replacement">
+                        <replacement>completionSizeExpression</replacement>
+                        <search>completionSize</search>
```

Comment:
If we do this search all compleTmeout chunks will be replaced, so if the file already has completionSizeExpression the result will be : completionSizeExpressionExpression